src/logger.js

```diff
@@ -14,13 +15,17 @@ function defaultLogger() {
   return new LoggerController(adapter, null, options);
 }
 
-let logger = defaultLogger();
+let logger;
+const defaultLogger = basicLogger();
 
 export function setLogger(aLogger) {
   logger = aLogger;
 }
 
 export function getLogger() {
+  if (!logger) {
+    return defaultLogger();
```

Comment:
do you just want to instantiate the default logger once instead of instantiating a new instance for earch get logger call?

i.e. 

```
if (!logger) {
  logger = defaultLogger();
}
return logger;
```

Reply:
I just noticed that